index.js

```diff
@@ -4,7 +4,33 @@
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
  * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+    let arr = process.argv. slice(2);
+    let sum = 0;
+    
+    if (arr[0] === undefined) {
+        return "No operation provided...";
+    } else if (arr[0] !== 'minus' && arr[0] !== 'plus') {
+        return `Invalid operation: ${arr[0]}`
+    } else if (arr[1] === undefined) {
+         return ("No numbers provided...");
+    }
+
+    if (arr[0] === 'plus') {
```

Comment:
The code in this if block and the one on line 23 are very similar with few differences. How can you refactor this code to be less repetitive??